App.Web/Modules.cs

```diff
@@ -1,25 +1,23 @@
-﻿using App.Configuration;
-using App.Deposits;
+﻿﻿using App.Configuration;
 
 namespace App.Web
 {
     /// <summary>
     /// IMPORTANT ! In order to use classes and endpoints, defined in your own module, it should be referenced here as it shown
     /// </summary>
     [ModuleUsing(typeof(Example.ExampleModule))] // < ---- Example of module registration
+    [ModuleUsing(typeof(Users.UsersModule))]
 
     [ModuleUsing(typeof(News.NewsModule))]
     [ModuleUsing(typeof(UserSupport.UserSupportModule))]
     [ModuleUsing(typeof(Goods.GoodsModule))]
     [ModuleUsing(typeof(Stocks.StocksModule))]
     [ModuleUsing(typeof(Loans.LoansModule))]
-    [ModuleUsing(typeof(Cards.CardsModule))]
```

Comment:
Why cards and regularpayments modules were removed ?

Reply:
When i pull repository, cards and regular payments projects was installed wrong in my local instance and my visual studio version. That changes were made for successful project startup in my machine.

Reply:
@anton-beetroot-23071996 I did not find directory for App.RegualrPayments, but it is noticed in ApiTemplate.sln file. Visual studio cannot find sources for App.RegualrPayments. So application build fails. Lack of App.RegualrPayments sources i find in curent state of master branch in https://github.com/NikolenkoAnton/KPZ-PI-318-SUBGROUP_2 repository. Please pay attention.